firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -357,10 +357,24 @@ class BrowserViewController: UIViewController,
 
     @objc
     fileprivate func appMenuBadgeUpdate() {
-        let actionNeeded = RustFirefoxAccounts.shared.isActionNeeded
-        let showWarningBadge = actionNeeded
+        let isActionNeeded = RustFirefoxAccounts.shared.isActionNeeded
+        let showWarningBadge = consume isActionNeeded
```

Comment:
Yes, I'm interested in making a presentation about the `consume` keyword. In this example, by using `consume`, we explicitly tell the compiler to change the ownership of the variable instead of copying it, which improves performance. For more details, you can take a look to the proposal: https://www.linkedin.com/in/paraipansorin/recent-activity/all/